### PR TITLE
everything works

### PR DESCRIPTION
**Pull request by mmichalm**

### CODE REVIEW

File: module1/homework/calculate/calculate.hpp [hunk 1]
```diff
@@ -2,6 +2,43 @@
...
+    std::string result("0");
+	if(command == "add" || command == "subtract" || command == "multiply" || command == "divide")
+	{
+    char operation = command[0];
+	
+	switch (operation) 
+	{
+	case 'a':
+		result = std::to_string(first + second);
+		return result;
```
Comment (DamiXioR) [inline module1/homework/calculate/calculate.hpp:14]: in my opinion it's shorter to use directly return std::to_string(first + second);

File: module1/homework/calculate/calculate.hpp [hunk 2]
```diff
@@ -2,6 +2,43 @@
...
+		{
+			result = std::to_string(first / second);
+			return result;
+		}
+		else
+		{
+			
+			return "Division by 0"; 
+		}
+		break;
```
Comment (DamiXioR) [inline module1/homework/calculate/calculate.hpp:35]: i think this else is unnecessary, if case 'd' is last case label you do not need to use break statement (and it's just after last return - never execute)

File: module1/homework/calculate/calculate.hpp [hunk 3]
```diff
@@ -2,6 +2,43 @@
...
-    // TODO: Implement your solution here and return proper value
-    return "";
+    std::string result("0");
+	if(command == "add" || command == "subtract" || command == "multiply" || command == "divide")
+	{
+    char operation = command[0];
+	
+	switch (operation) 
+	{
+	case 'a':
```
Comment (piezol) [inline module1/homework/calculate/calculate.hpp:12]: That looks convenient 'for now' but requires major rework if list of commands ever changes.
Using switch doesn't really achieve anything here.

File: module1/homework/calculate/calculate.hpp (hunk 2 above)
Comment (piezol) [reply module1/homework/calculate/calculate.hpp:35]: @DamiXioR is correct, return exits function body alltogether.

**Comment by ziobron** [comment]
{ "requirement_ids": [1119, 1429], "points": [2, 5] }

**Pull request closed without merging**